meetings/2017-11-30.md

```diff
@@ -0,0 +1,122 @@
+# Node.js Foundation Community Committee (CommComm) Meeting 2017-11-30
+
+## Links
+
+
+* **Recording**:  https://www.youtube.com/watch?v=DtYASTx99WM 
+* **GitHub Issue:** https://github.com/nodejs/community-committee/issues/168 
+* **Minutes Google Doc:** https://docs.google.com/document/d/1SWJjGXEpTdu_2RbUkHr05h3WZco3pFrq59p4gDoKTnc/edit?usp=sharing
+<https://docs.google.com/document/d/1SWJjGXEpTdu_2RbUkHr05h3WZco3pFrq59p4gDoKTnc/edit?usp=sharing>
+* **Previous Minutes Google Doc:** <https://docs.google.com/document/d/1-oCbwBTLm7yK47hc5iMCXcR78n6L1g4l2-VuRYQm5CM/edit?usp=sharing>_
+
+
+## Present
+Michael Dawson (@mhdawson - CommComm)
+Tierney Cyren (@bnb - CommComm)
+Tracy Hinds (@hackygolucky - CommComm)
+Olivia Hugger (@pup - CommComm)
+Ben Michel (Observer ?)
+Dan Shaw (Obverser, End user feedback initiative)
+Jem bezooyen (Observer)
+Manil (Observer)
+Nikhil Komawar (Observer)
+Pooya Paridel (Observer)
+Rafel Ackermann(@refack CommComm)
+Ahmad Bamieh (Observer)
+Benjamin Zaslavsky (@tiriel - Observer)
+
+
+
+## Notes
+
+Extracted from cc-agenda labelled issues and pull requests from the nodejs org prior to the meeting.
```

Comment:
all of this needs to be formatted as bullet points, markdown ignores single line breaks